libs/domain/site/src/models/option.model.ts

```diff
@@ -1,4 +1,4 @@
-export interface Salutation {
+export interface Option {
```

Comment:
this is considered a breaking change

Reply:
Should we avoid it and create the same Gender interface or just describe this BC in PR description?

Reply:
If we want to move away from the current design, we should:
- keep Salutation as-is, but deprecate it
- introduce new Option
- ensure code can still work with Saluation, but use Option as the main model going forward